tests/integration/test_notebooks_gpu.py

```diff
@@ -264,14 +264,14 @@ def test_slirec_quickstart_integration(
     "epochs, seed, expected_values",
     [
         (
-            10,
+            8,
```

Comment:
Any reason we changed test epochs in these tests?

Reply:
it finished earlier, not special reason